device/accton/x86_64-accton_as4630_54te-r0/sonic_platform/thermal.py

```diff
@@ -0,0 +1,143 @@
+#############################################################################
+# Edgecore
+#
+# Thermal contains an implementation of SONiC Platform Base API and
+# provides the thermal device status which are available in the platform
+#
+#############################################################################
+
+import os
+import os.path
+import glob
+
+try:
+    from sonic_platform_base.thermal_base import ThermalBase
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
+
+
+class Thermal(ThermalBase):
+    """Platform-specific Thermal class"""
+
+    THERMAL_NAME_LIST = []
+    SYSFS_PATH = "/sys/bus/i2c/devices"
+
+    def __init__(self, thermal_index=0):
+        self.index = thermal_index
```

Comment:
Please call the superclass initializer here for completeness.

Reply:
Thanks for review, I'll modify it.